app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module InPerson
+  class SendProofingNotificationJob < ApplicationJob
+    # @param [Number] enrollment_id primary key of the enrollment
+    def perform(enrollment_id)
+      return if IdentityConfig.store.in_person_proofing_enabled.blank? ||
```

Comment:
wondering again if we need to do these checks here in this job since we check in the getUspsProofingResultsJob before starting this one

Reply:
@svalexander , this may be useful when manually invoke the job.

Reply:
good point! that makes sense then